project/AppDependencies.scala

```diff
@@ -6,34 +6,34 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.2.0-play-26",
+    "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.3.0-play-26",
     "uk.gov.hmrc"          %% "logback-json-logger"           % "4.8.0",
     "uk.gov.hmrc"          %% "govuk-template"                % "5.55.0-play-26",
     "uk.gov.hmrc"          %% "play-health"                   % "3.15.0-play-26",
     "uk.gov.hmrc"          %% "play-ui"                       % "8.11.0-play-26",
-    "uk.gov.hmrc"          %% "bootstrap-play-26"             % "1.13.0",
+    "uk.gov.hmrc"          %% "bootstrap-frontend-play-26"    % "2.24.0",
```

Comment:
What's this? I think this library need to be without frontend

Reply:
This is some new approach that has been taken: https://github.com/hmrc/bootstrap-play

Reply:
That's great, good to know